tests/skipjob/cron-job-timezone/skipjob.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: SKIPJob
+metadata:
+  name: cron-job-timezone
+spec:
+  container:
+    image: "perl:5.34.0"
+    command:
+      - "perl"
+      - "-Mbignum=bpi"
+      - "-wle"
+      - "print bpi(2000)"
+  cron:
+    schedule: "* * * * *"
+    timeZone: "Europe/Oslo"
```

Comment:
i think you tested this locally, but it might be good to change this to an uncommon timezone just to avoid false positives